File: examples/comparison/fib/fib.go
package main

import "fmt"

func main() {
	const steps int64 = 375_000_000
	const mod int64 = 1_000_000_007
	var a, b int64 = 0, 1
	for i := int64(0); i < steps; i++ {
		t := (a + b) % mod
		a = b
		b = t
	}
	fmt.Println(b)
}

File: examples/comparison/fib/fib.js
const STEPS = 375_000_000;
const MOD = 1_000_000_007;
let a = 0;
let b = 1;
for (let i = 0; i < STEPS; i++) {
  const t = (a + b) % MOD;
  a = b;
  b = t;
}
console.log(b);
